tests/foreman/api/test_user.py

```diff
@@ -6,19 +6,19 @@
 """
 import ddt
 from fauxfactory import gen_string
+from nailgun import client
 from random import randint
 from requests.exceptions import HTTPError
-from robottelo import entities
+from robottelo.common.helpers import get_server_credentials
 from robottelo.common import decorators
+from robottelo import entities
```

Comment:
@elyezer, thanks for the pointer. I didn't know that vim has a built-in sort. That helps.